webui/react/src/utils/storage.ts

```diff
@@ -76,11 +81,44 @@ export class Storage {
     this.store.setItem(path, item);
   }
 
+  keys(): string[] {
+    const prefix = this.pathKeys.join(this.delimiter) + this.delimiter;
+    return this.store.keys()
```

Comment:
**question:** why not just use `Object.keys(this.store)` here and avoid having to prototype?

Reply:
the idea is that this gives each store a chance to implement their own `.keys`. However looking at the two store we currently have what you're suggesting would work as well. should I update it?

Reply:
I see, no how you have it is good.